docs/src/content/docs/guides/customer-support.mdx

```diff
@@ -16,11 +16,11 @@ In these cases, you'll need your support team to help!
 
 ## The Example on GitHub
 
-The code for this example can be found [in our examples repo](https://github.com/buttonize/examples/tree/master/customer-support).
+The code for this example can be found [in our examples repo](https://github.com/buttonize/buttonize/tree/master/examples/customer-support).
 Please check it out and get acquainted.
 
 One import thing to note is that in order to use the local development tooling with Buttonize, you _must_ export your CDK app,
-as can be seen [here](https://github.com/buttonize/examples/blob/3c0afb1bc0e9c48bd6ef0d22e487b3a0832e4975/customer-support/bin/customer-support.ts#L6).
+as can be seen [here](https://github.com/buttonize/buttonize/blob/ec970fccbbf42b8c04d439ca6c63f321dbb4174b/examples/customer-support/bin/customer-support.ts#L8).
```

Comment:
Let's add links directly pointing to `master` branch, not the commit. So that the future updates are reflected.